### DISCUSSION TITLE
Change disk to S3

### DISCUSSION THREAD

**Discussion by gotzsys** [pxlrbt/filament-excel#202] [General]
How can I change the disk location to s3? I tried adjusting the configuration in the service provider boot as shown here: https://github.com/pxlrbt/filament-excel/issues/152

```
config()->set('filesystems.disks.filament-excel', [
    'driver' => 'your_driver',
    // other configuration options
]);
```
However, it returns the error below, and my disk configuration is correct, as I’m using it for other uploads:

[2024-09-18 19:17:54] local.ERROR: Missing required client configuration options:

region: (string)

A "region" configuration value is required for the "s3" service
(e.g., "us-west-2"). A list of available public regions and endpoints can be
found at http://docs.aws.amazon.com/general/latest/gr/rande.html. {"exception":"[object] (InvalidArgumentException(code: 0): Missing required client configuration options:

region: (string)

A \"region\" configuration value is required for the \"s3\" service
(e.g., \"us-west-2\"). A list of available public regions and endpoints can be
found at http://docs.aws.amazon.com/general/latest/gr/rande.html. at /var/www/html/vendor/aws/aws-sdk-php/src/ClientResolver.php:1356)
[stacktrace]

**Comment by pxlrbt** [comment]
What's the actual code you use? Did you set the `region` as the error message suggests?
